arduino/smartcar/smartcar.ino

```diff
@@ -14,18 +22,39 @@ void setup() {
 }
 
 void loop() {
-  car.setSpeed(90);
+  car.setSpeed(70); 
   preventCrash();
 }
 
-void preventCrash() {
-  unsigned int fDistance = front.getDistance();
-  if (fDistance > 0 && fDistance < 200) {
+void preventCrash(){
+  preventCrashWithUltrasonic(); 
+  preventCrashWithInfrared(frontIR, 0, 40); // Helps in short distances object detection.
+}
+
+void preventCrashWithUltrasonic() {
+  unsigned int distanceUS = frontUS.getDistance(); // min distance: 0.2 m, MAX distance: 4 m.
+  if (distanceUS > 0 && distanceUS < 200) {
```

Comment:
Comment in line 35: Isn't the max distance set to 200 in line 7 and does that not correspond to 2m? The if statement also does not reflect the comment, checking between values 0 and 200, rather than 20 and 400 (unless 200 is 4m).
I find it questionable to avoid obstacles at a 4m margin though. The 2m I picked because it let the car prevent the crash even at 100% speed.

Reply:
Typo fixed. Thanks!